examples/ColorCamera/rgb_camera_control.py

```diff
@@ -2,21 +2,41 @@
 
 """
```

Comment:
Sometime later would be good to update the C++ example too.

Reply:
cc @moratom to match python example:)